src/pages/version/ManifestBlob.tsx

```diff
@@ -0,0 +1,43 @@
+import { StyledLink } from "components/styles";
+import { P2 } from "components/Typography";
+import { string } from "utils";
+
+const { omitTypename } = string;
+// typescript interface with manifest that is an object of key-value pairs
+interface Props {
+  manifest: {
+    id: string;
+    revision: string;
+    project: string;
+    branch: string;
+    isBase: boolean;
+    moduleOverrides?: {
+      [key: string]: string;
+    };
+    modules?: {
+      [key: string]: {
+        [key: string]: string;
+      };
+    };
```

Comment:
Did you enumerate this instead of importing from `gql/generated/types` in order to have more precision over the `modules` and `moduleOverrides` type?

Reply:
Thats a good point. That was my reason for defining the type, but you are correct we don't actually do anything with it apart from reexporting that data so it doesn't make sense to redeclare the type. I updated it to just use the generated type